docs/groups.md

```diff
@@ -74,6 +74,7 @@ Read more in [Bucketing](/docs/bucketing).
 
 - A feature can only belong to a maximum of one group at a time
 - A feature cannot repeat in the same group's slots (this will be supported in a future version)
+- Both the parent and child features cannot coexist in the same group
```

Comment:
nothing to do with this line, but I have to admit that when I read "groups" the first thing that came up to my mind was "the parents feature above could also be called groups because we're just grouping features and enabling them all at once". Have you considered to call them groups and renaming groups (which are technically mutually exclusive only- groups)? Or even wilder idea, can both be called groups and we've just got different groups behaviours?

Reply:
with groups: 

- features need to be aware of each other (bidirectional), as they occupy the slots of a group together without overlapping

with parents/required:

- only the dependents need to be aware of the primary/parent/dependency feature, so it's more unidirectional

---

if both cases were following the same direction, either only bidirectional or only unidirectional, it would have been easier to have the core Groups concept applied to both cases.

but given Groups are there for slot assignments only, it doesn't fit parents/required model because they don't care or have anything to do with slots and overlapping concerns.